config/core/resources/apiserversource.yaml

```diff
@@ -79,5 +79,5 @@ spec:
     served: true
     storage: true
   - name: v1alpha2
-    served: false
+    served: true
     storage: false
```

Comment:
I'm confused between this and the comments. Shouldn't this be flipped to storage = true?

Reply:
it is stored as v1alpha1 still.

Reply:
it is lossy so I think we have to do that manual upgrade touch thing. So I am leaving it as v1alpha1 stored until we fix this issue as a whole